_toc.yml

```
@@ -0,0 +1,17 @@
+format: jb-book
+root: README
+parts:
+  - caption: Relevant Research Papers
```

```suggestion
  - caption: Previous Work Done
```
Would it make sense to keep caption names a bit broad?